index.bs

```diff
@@ -2140,6 +2251,16 @@ that the client knows what the server certificate is or how to contact it is
 not sufficient.  The application has to establish the identity of the client
 in-band if necessary.
 
+## Pooled Sessions
+
+When pooling, if information from one session is affected by the activity of another
+session, one session could infer information about the activity of another session.
+For example, if the rate control feedback of one session is affected by the rate at
+which an application is sending data on another session, one session could learn about the
+the rate at which another application is sending data.  To avoid this, a user agent
+can restrict access to information or modify information provided.  For example, it could leave
+the send rate at an artifical value to make it more difficult to infer anything from it.
```

Comment:
This seems pretty loose.  This boils down to "maybe the browser could work something out", which is pretty unsatisfying.  We should be more concrete, either block access to the attribute (which reveals the presence of another entity, so that needs thought) or define an exemplar strategy for making it available.